test/unit/test-SqlString.js

```diff
@@ -25,6 +26,15 @@ test('SqlString.escapeId', {
     assert.equal(SqlString.escapeId(['a', 'b', 't.c']), "`a`, `b`, `t`.`c`");
   },
 
+  'sets are turned into lists': function() {
+    if (!iterableSupport) return;
```

Comment:
Typically it is best that if a test is going to be skipped, it's simply just not added to the object so it doesn't show up like there is a test happening. At least, the pattern with `utest`, which does not have a skip mechanism.